go/store/prolly/message/message.go

```diff
@@ -35,27 +35,18 @@ type Serializer interface {
 	Serialize(keys, values [][]byte, subtrees []uint64, level int) serial.Message
 }
 
-func UnpackFields(msg serial.Message) (keys, values ItemAccess, level, count uint16, err error) {
-	switch serial.GetFileID(msg) {
+func UnpackFields(msg serial.Message) (fileId string, keys, values ItemAccess, level, count uint16, err error) {
+	fileId = serial.GetFileID(msg)
+	switch fileId {
 	case serial.ProllyTreeNodeFileID:
-		return getProllyMapKeysAndValues(msg)
+		keys, values, level, count, err = getProllyMapKeysAndValues(msg)
```

Comment:
Kind of a nit but I prefer explicit immediate returns instead of implicit ones where possible, it's very easy to mistakenly alter the values of these vars in between assignment and return otherwise. (You're also missing an explicit return in the BlobFileId case)

Reply:
Oh, I agree. I would have loved it if Go would let me write, for instance:

```go
return fileId, getProllyMapKeysAndValues(msg)...
```

But since that's not supported, we have to do the assignment regardless.